### DISCUSSION TITLE
How to add multiple Google Clients to one self-hosted instance

### DISCUSSION THREAD

**Discussion by thimplicity** [plausible/analytics#2909] [Self-Hosted Support]
Hi, I am hosting plausible via docker and have two websites that I monitor with it. I have connected the Google Search API to one of them through the official documentation that instructs one to add a google client ID and secret as environment variables. I would like to add the second website's Search API (different Google account) to the plausible instance as well.

How is that possible?
Thanks in advance for some guidance!

**Comment by wheeleran** [comment]
If you add one Google account as a Full User or even co-owner on the Search Console site of another, can you pull the data from both?

**Reply by thimplicity** [reply]
Good idea, will try

**Reply by thimplicity** [reply]
Did not work unfortunately - any other ideas?

**Reply by wheeleran** [reply]
unfortunately not, and there are a number of 'Search Console' tickets open, some more than a year, and one from me too. i'm starting to doubt that it's currently worth it. i've got all my sites under one google account so i don't have your problem, but tokens expire after 7 days anyway - testing OR live app - so you have to go through and re-link all your sites one by one which is tedious. then there's the unresolved ```invalid_grant``` error, which may or may not be related. @Tony-MF says it works for him, but i don't know what he may or may not have done differently. good luck.
